content/gotraining.slide

```diff
@@ -212,21 +224,265 @@ Execute with
 * Interlude: Towards Continuous Deployment
 
 
+* What is Continuous Deployment (CD)?
+
+Continuous Deployment != Continuous Delivery
+
+.image img/cd.jpg
+.caption [[https://blog.crisp.se/2013/02/05/yassalsundman/continuous-delivery-vs-continuous-deployment][Yassal Sundman, Continuous Delivery vs Continuous Deployment]]
+
+
+* Continuous Deployment Quotes
+
+"Continuous Deployment is a strategy for software releases wherein any code commit that passes the automated testing phase is automatically released into the production environment."
+[[https://searchitoperations.techtarget.com/definition/continuous-deployment][TechTarget definition]]
+
+
+"Continuous deployment requires a highly developed culture of monitoring, being on call, and having the capacity to recover quickly."
+[[https://semaphoreci.com/blog/2017/07/27/what-is-the-difference-between-continuous-integration-continuous-deployment-and-continuous-delivery.html][Marko Anastasov, Semaphore Engineering Blog]]
+
+
+* Recommendations for CD @ANZ (1/2)
+
+- Use *trunk* *based* *development* (abandon GitFlow)
```

Comment:
there's a lot of writing about trunk based dev here: https://paulhammant.com/categories.html#Trunk-Based_Development